resources/codegen_plan.toml

```diff
@@ -13,6 +13,11 @@ mode = "parse"
 source = "resources/codegen_inputs/colr.rs"
 target = "read-fonts/generated/generated_colr.rs"
 
+[[generate]]
+mode = "parse"
+source = "resources/codegen_inputs/cpal.rs"
+target = "read-fonts/generated/generated_cpal.rs"
+
 [[generate]]
```

Comment:
Do you also need a block with mode = "compile"? I have one in my PR.

Reply:
Hm. It observably compiles but ... that does seem consistent with others. Curiously colr doesn't have one.

Reply:
Fixed

Reply:
fwiw: the 'compile' entry is only necessary if you are also adding support for the module in `write-fonts`.